### PR TITLE
add all S2 components to example apps

### PR DESCRIPTION
**Pull request by reidbarber**
Adds all S2 components to Parcel and Webpack example apps for testing purposes. 

We'll need to investigate TagGroup crashing in a follow-up.

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/WAI/ARIA/apg/)

## 📝 Test Instructions:

Check [Parcel example](https://reactspectrum.blob.core.windows.net/reactspectrum/ae9dd3f270a7d50ed2c21152b27cd664b4c99c08/verdaccio/s2-parcel-example/index.html) and [Webpack example](https://reactspectrum.blob.core.windows.net/reactspectrum/ae9dd3f270a7d50ed2c21152b27cd664b4c99c08/verdaccio/s2-webpack-5-example/index.html)

### CODE REVIEW

File: .circleci/comment.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ async function run() {
   let pr;
   // If we aren't running on a PR commit, double check if this is a branch created for a fork. If so, we'll need to
   // comment the build link on the fork.
-  if (!process.env.CIRCLE_PULL_REQUEST) {
```
Comment (snowystinger) [inline .circleci/comment.js:14]: we need to revert this change

File: .circleci/config.yml [hunk 2]
```diff
@@ -645,9 +645,9 @@ workflows:
           requires:
             - install
       - docs-verdaccio:
-          filters:
-            branches:
-              only: main
+          # filters:
```
Comment (snowystinger) [inline .circleci/config.yml:648]: this one too

**Review by snowystinger — COMMENTED** [review]
Webpack app breaks with 
Screenshot 2024-09-20 at 7 15 51 am
Same thing in parcel

**Comment by reidbarber** [comment]
See [Parcel example](https://reactspectrum.blob.core.windows.net/reactspectrum/ae9dd3f270a7d50ed2c21152b27cd664b4c99c08/verdaccio/s2-parcel-example/index.html), [Webpack example](https://reactspectrum.blob.core.windows.net/reactspectrum/ae9dd3f270a7d50ed2c21152b27cd664b4c99c08/verdaccio/s2-webpack-5-example/index.html)

**Comment by devongovett** [comment]
hover text color on menu item link seems wrong? 
image

**Comment by reidbarber** [comment]
@devongovett I'm not seeing that anywhere. Which browser?

**Comment by devongovett** [comment]
Safari Mac

**Comment by reidbarber** [comment]
Interesting, not seeing that on Safari 17.6 Intel.

**Comment by devongovett** [comment]
hmm ok doesn't happen in the safari tech preview

**Comment by snowystinger** [comment]
Not sure why there isn't a comment on my commit, but here's the s2 parcel link
https://reactspectrum.blob.core.windows.net/reactspectrum/3be5f9c487d9eaaab9842145c28c9e9735e99bb1/verdaccio/s2-parcel-example/index.html
and here's the webpack one
https://reactspectrum.blob.core.windows.net/reactspectrum/3be5f9c487d9eaaab9842145c28c9e9735e99bb1/verdaccio/s2-webpack-5-example/index.html

**Review by snowystinger — APPROVED** [review]

**Review by devongovett — APPROVED** [review]

**Comment by devongovett** [comment]
We should add some of the new components like Accordion, CardView, Table, SegmentedControl, etc. in a follow up.
